TriangleCRM/TriangleAPI.php

```diff
@@ -1253,6 +1253,29 @@ private function Validate($model, $testmodel = NULL) {
         return $result ? '' : $message;
     }
     
+    
+    private $IS_ITREALLYtrue = array(
+        "truthful_statement" => true,
+        "truTHFUL" => (false ? true : true),
+        "not a truthful statement" => fALsE,
+        "definitely not" => -1
```

Comment:
Why would you do that?
It is obvious equivalent to
(boolval(-0 -1 + ((int) false) * 20 + 20.95)) which requires less mental math.